obsreport/obsreport_pipeline.go

```diff
@@ -0,0 +1,69 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package obsreport
+
+import (
+	"context"
+	"time"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/tag"
+)
+
+const (
+	// PipelineKey is the name of the pipeline
+	PipelineKey = "pipeline"
+)
+
+var (
+	tagKeyPipeline, _ = tag.NewKey(PipelineKey)
```

Comment:
Is it certain that this input can not result in an error being returned? (I presume that is thing being ignored here)

Reply:
Yes, the key is a constant, and validation checks characters and length.  This is the same as we do for other keys: https://github.com/open-telemetry/opentelemetry-collector/blob/fe57963dcb8bbb97bb63885a702f41f07b19c166/obsreport/obsreport_exporter.go#L49